primedev/squirrel/squirrel.cpp

```diff
@@ -756,9 +756,6 @@ ON_DLL_LOAD_RELIESON("client.dll", ClientSquirrel, ConCommand, (CModule module))
 
 	StubUnsafeSQFuncs<ScriptContext::CLIENT>();
 	StubUnsafeSQFuncs<ScriptContext::UI>();
-
-	g_pSquirrel<ScriptContext::CLIENT>->__sq_getfunction = module.Offset(0x6CB0).RCast<sq_getfunctionType>();
-	g_pSquirrel<ScriptContext::UI>->__sq_getfunction = g_pSquirrel<ScriptContext::CLIENT>->__sq_getfunction;
```

Comment:
Why is the UI one not getting set anymore

Reply:
it still [is](https://github.com/R2Northstar/NorthstarLauncher/blob/150f008871642daeb24591566e288865ecdc82bf/primedev/squirrel/squirrel.cpp#L718C1-L719C106).
I just removed this duplicate assignment.